meta-genivi-dev/poky/meta/recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -1,5 +1,2 @@
-FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+FILESEXTRAPATHS_prepend := ":${THISDIR}/${PN}"
```

Comment:
As this is a prepend the colon should be at the end. Yocto is warning about this when building for r-car-m3-starter-kit